marketplace/scripts/codeExampleInliner.ts

```diff
@@ -0,0 +1,135 @@
+import path from 'path';
+import fs from 'fs';
+
+const PATH_TO_EXAMPLES = path.resolve('../examples');
+const CODE_EXAMPLE_START_REGEX = /<(CodeExample|CliInvocationExample)/g;
+const CODE_EXAMPLE_PROP_REGEX = /([a-zA-Z0-9]+)={?["`](.*?)["`]}?/g;
+
+export type CodeExampleFormatProps = {
+  lineStart?: string;
+  lineEnd?: string;
+  startAfter?: string;
+  endBefore?: string;
+  dedent?: string;
+};
+
+export async function inlineCodeExampleFileReferences(content: string) {
+  const codeExampleMatches: {
```

Comment:
This approach is the same as before, but I broke the `CODE_EXAMPLE_PATH_REGEX` into two regexs, because rendering the code examples correctly requires parsing all the props, not just the path, and because the order of the props can vary which broke the old regex.